src/main/java/de/tum/in/www1/artemis/repository/SubmissionRepository.java

```diff
@@ -26,6 +26,9 @@
     @Query("select submission from Submission submission where type(submission) in (ModelingSubmission, TextSubmission) and submission.submitted = false and not submission.participation is null")
     List<Submission> findAllUnsubmittedModelingAndTextSubmissions();
 
+    /* Get all submissions from a participation_id */
+    List<Submission> findAllByParticipationId(Long participationId);
```

Comment:
As discussed: You should be able to reuse `findByParticipationId`.

Reply:
After reading up on it, we should rather remove the `findByParticipationId` method.
I created a follow up for this task. #760

Reply:
I am not sure how this should work. You also display the result of a submission, however this is fetched lazily in the database. When and where do you load each result of each submission? I think you should do that here and not load them again in the client in single calls. Please double check this.

Reply:
I am not 100% sure, but this should solve it:
@EntityGraph(attributePaths = { "result" })
    List<Submission> findAllByParticipationId(Long participationId);

Reply:
> After reading up on it, we should rather remove the `findByParticipationId` method.
> I created a follow up for this task. #760

You mean to remove findAllByParticipationId, correct? Because the All doesn't do anything.